SeleniumTestXUnit/Tests/Features/Project/DragAndDrop.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Project Interacting
```

Comment:
Perhaps a better name for the feature could be "Project drag and drop actions"

Reply:
Done